cli/crates/cli/src/start.rs

```diff
@@ -62,7 +64,15 @@ where
 
     let handle = thread::spawn(move || {
         #[allow(clippy::ignored_unit_patterns)]
-        tokio::runtime::Runtime::new().unwrap().block_on(async {
+        let runtime = tokio::runtime::Runtime::new().unwrap();
+        runtime.block_on(async {
```

Comment:
The new gateway is in another castle. We have a crate `production-server` where we need to add this too when we merge my PR.